### DISCUSSION TITLE
7 Unique Cloners for my Repo?

### DISCUSSION THREAD

**Discussion by echorashmi** [community/community#22069] [New to GitHub]
<p>So I’ve just created a repo and I’m making commits regularly as I continue building github into my workflow.</p>
<p>I just looked at the traffic tab today and saw there were “7 Unique cloners” on my repository.</p>
<p>Are actions that I take on github (new branch, merge, push, pull) counted towards these clones?</p>
<p>If it is real people, I’m curious why there would be anyone cloning my repo, it’s pretty basic C stuff.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/echorashmi">@echorashmi</a>! :wave: Welcome to the Community!</p>
<p>Those unique cloners really are separate people, yeah! (Well, unless you made a new branch on one machine, and then swapped to a different computer to merge, and then a different one to push, but I’m guessing you’re not doing that! :laughing: )</p>
<p>It’s very common to have 6 or 7 unique cloners on almost every public repo, because there are several projects out there that routinely clone all public repos. Some organizations do it to scan for API keys or other credentials accidentally made public.</p>
<p>There are also projects that clone everything for archival purposes, such as:</p>
<p><a href="http://ghtorrent.org/" class="onebox" target="_blank">http://ghtorrent.org/</a></p>
<p><a href="https://www.gharchive.org/" class="onebox" target="_blank">https://www.gharchive.org/</a></p>

**Reply by sikandermukhtar** [reply]
Informative

**Reply by My-Computer-Is-Trash** [reply]
What 0 website visitations, and 0 repository visitations until 7 days after my repository was created, when there's 21 unique git clones? Like, for what reason, are 21 different programs, saving and logging my website and code? I have a GNU licence so I'm not actually worried about anything. However, it still makes me kind of paranoid...

![Screenshot 2023-05-04 9 11 17 AM](https://user-images.githubusercontent.com/91203391/236214592-ffca382d-f7b6-4888-8aa4-829f70b969a1.png)

**Reply by MachineHerald007** [reply]
Yes, and others are more nefarious than just API key scanning. A lot also scan repos to see if they have vulnerabilities through common malpractices or if they're using known, unsafe packages-- these are only issues if you're hosting the code somewhere.

If you know what you're doing, you shouldn't have anything to worry about.
